src/utils/common.ts

```diff
@@ -694,3 +696,27 @@ export function decryptedComputed(
       set: setEncryptedState,
     });
 }
+
+export async function exportFile(
+  text: string,
+  filename: string,
+): Promise<string | undefined> {
+  const blob = new Blob([text], { type: 'text/plain' });
+  const a = document.createElement('a');
+  const href = window.URL.createObjectURL(blob);
+
+  if (Capacitor.isNativePlatform()) {
```

Comment:
Should probably work fine with `IS_MOBILE_APP`. I didn't create this method, but from what I can tell, it will have the same effect, since we set the `PLATFORM` as `ionic` when building for iOS/Android.